models/marts/core/fct_ga4__pages.sql

```diff
@@ -0,0 +1,45 @@
+with page_view as (
+    select
+        event_date_dt,
+        extract( hour from (select  timestamp_micros(event_timestamp))) as hour,
+        page_location,  -- does not include query string; disable this model, copy this to your project and switch to original_page_location to segment by page_location with query string
```

Comment:
responding to your comment on this line... `page_location` does include the query string. The user can optionally strip specific query parameters from the query string using the `query_parameter_exclusions` var, but again, that only strips specific query string params. 

So I'm not sure the comment is needed, in fact you may want to say "-- page_location includes any query string params that were not filtered out by the query_parameter_exclusions variable"